### DISCUSSION TITLE
guys was there any change in the [Jan 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4808] [Discord Threads]
> 🙋‍♂️ **vikiiingur**

**Comment by o1lab** [comment]
> 🙋‍♂️ **alex.pro**  
#### <@901021819908263967> did you solve this issue? <@310157751395287040> I have the same result since 0.99 ...When I downgrade version to 0.98.4 everything works perfect  
#  
> 🙋‍♂️ **alex.pro**  
#### it is the same for 0.100.2  
#  
> 🙋‍♂️ **alex.pro**  
#### <@310157751395287040> How can I help you to investigate the issue?   
#### Node: **v16.17.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### Database: **mysql2**   
#### ProjectOnRootDB: **true**   
#### RootDB: **mysql2**   
#### PackageVersion: **0.98.4**  
#  
> 💁 **o1lab**  
#### did you guys try hardrefresh ?  
#  
> 💁 **o1lab**  
#### <@901021819908263967> : what database are you using ?  
#  
> 💁 **o1lab**  
#### <@939491479599067156> : what mysql database version is it ?  
#  
> 🙋‍♂️ **alex.pro**  
#### pranavxc helped me ... there were 2 errors   
#### the first one is fk_user_id field was in nc_api_tokens table, so I had to remove it   
#### the second one: collations were different in nc_api_tokens and nc_users_v2  
#  
> 🙋‍♂️ **alex.pro**  
#### thanks a lot <@682854952657616902>  <@310157751395287040>  
#  
> 🙋‍♂️ **vikiiingur**  
#### sry, just saw this, I am using postgres as db  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1034065936631730247)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
